.readthedocs.yml

```diff
@@ -0,0 +1,22 @@
+---
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python in the build environment.
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.10"
+
+mkdocs:
+  configuration: "mkdocs.yml"
+  # fail_on_warning: true
```

Comment:
I recommend uncommenting this as it forces resolution of all mkdocs warnings - they are generally useful.